lib/gui/models/store.js

```diff
@@ -46,6 +48,11 @@ const DEFAULT_STATE = Immutable.fromJS({
     unmountOnSuccess: true,
     validateWriteOnSuccess: true,
     sleepUpdateCheck: false,
+
+    // Only set this by default if the installed
+    // version is not a production version
+    includeUnstableUpdateChannel: release.getReleaseType(packageJSON.version) !== release.RELEASE_TYPE.PRODUCTION,
```

Comment:
Just imagine them as perpendicular axes in some multi-dimensional co-ordinate system ;-)

On one axis you have stable / unstable (determined by `.isStableRelease`) and on another axis you have snapshot / production (determined by `.getReleaseType`).